src/lib/vendor/jquery.smooth-scroll.js

```diff
@@ -0,0 +1,357 @@
+/*!
+ * jQuery Smooth Scroll - v2.2.0 - 2017-05-05
```

Comment:
Could we also credit Smooth Scroll (by including this header) in our existing LICENSE file?